src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,81 @@
+.cards__card {
+  &:nth-child(n) {
+    margin-bottom: 46px;
+  }
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+}
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  width: get-width(200px, 4, 48px);
+  margin: 0 auto;
+  padding: 50px 0;
```

Comment:
why do you need a `cards `block? You can do the same with `BEM `mix. And remember that you can't add external styles (positioning or margins) to BEM-blocks